src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -526,6 +543,48 @@ abstract class DatabaseTest {
     return row.id!!.also { inserted.projectIds.add(it) }
   }
 
+  private var nextDeliverableNumber: Int = 1
+
+  protected fun insertDeliverable(
+      createdBy: UserId = currentUser().userId,
+      createdTime: Instant = Instant.EPOCH,
+      deliverableCategoryId: DeliverableCategory = DeliverableCategory.FinancialViability,
+      deliverableTypeId: DeliverableType = DeliverableType.Document,
+      descriptionHtml: String? = "",
+      id: DeliverableId = nextDeliverableNumber.toIdWrapper { DeliverableId(it) },
```

Comment:
A couple things here. The usual pattern is to make the ID nullable and let the database allocate an ID if the caller doesn't pass one in. And also, the ID arguments are usually of type `Any?` such that a caller that wants specific IDs can just use integer arguments like `id = 1` for brevity.